lvx frontend/src/App.js

```diff
@@ -2,8 +2,24 @@ import React from "react";
 import Navbar from "./Components/Navbar/Navbar";
 import Item from "./Components/Item/Item";
 import data from "./data.json";
+import { useEffect } from "react";
 import "./App.css";
 function App() {
+
+  const url = 'http://127.0.0.1:2000/lvx/items'
```

Comment:
Add `http://127.0.0.1:2000` in `.env` file and use `process.env` here.

Reply:
This way when you are deploying the application it becomes easier to change url in one file.